.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish:
+    name: Build and publish to PyPI
+    runs-on: ubuntu-18.04
```

Comment:
Any reason to use such an old version?
Using `ubuntu-latest` (the latest LTS version) would probably be ok.

Reply:
I just copied it over from the linked tutorial. To be honest, I would actually prefer to have the version locked, as that should be guaranteed to always work, while a dynamic version might introduce unexpected (breaking) changes.

Ultimately, it doesn't really matter, so I changed it to `ubuntu-latest`.

We do also list 3.8 as our preferred python version in the Pipfile.

Reply:
> To be honest, I would actually prefer to have the version locked, as that should be guaranteed to always work, while a dynamic version might introduce unexpected (breaking) changes.

I agree. In that case I would suggest to lock it to Ubuntu 20.04 (the current LTS).
The new LTS will be 22.04, which is coming out this year and 18.04 support ends next year. But I think there would be any breaking changes related to the things we need it to do in these GitHub workflows, so `ubuntu-latest` is fine.

Resolving